src/form-builder/components/FormDetailContainer.jsx

```diff
@@ -177,11 +175,23 @@ export class FormDetailContainer extends Component {
         uuid: formResourceUuid,
       };
       this._saveFormResource(formResource);
-      this._saveFormPrivileges(
+
+      if (this.state.formPrivileges.length === 0) {
+        let version = this.state.formData.version;
+        version--;
```

Comment:
@buvaneswari-arun can you please check the line formatting here

Reply:
This piece is removed due to revamp of code.